build_packages.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# MIT License
+#
+# Copyright (c) 2022-2024 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -e
+
+PACKAGE_NAME="ROCm-SDK-Examples"
+PACKAGE_VERSION="6.2.0"
+PACKAGE_VENDOR="Advanced Micro Devices, Inc."
+PACKAGE_CONTACT="ROCm Developer Support <rocm-dev.support@amd.com>"
+PACKAGE_DESCRIPTION_SUMMARY="A collection of examples for the ROCm software stack"
+PACKAGE_INSTALL_PREFIX="/opt/rocm/examples"
+
+BUILD_DIR="$(pwd)/build"
```

Comment:
```suggestion
GIT_TOP_LEVEL=$(git rev-parse --show-toplevel)
BUILD_DIR="$(GIT_TOP_LEVEL)/build"
```
To help the script find the top level.